recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "1.3.0.post" %}
-{% set commit = "b69669a230e59e36a193b679fc2c5b4d9150f8c9" %}
-{% set sha256sum = "984b02757e80117597a640cf3a1e7a66eeee091aefc1067458ca4251f0e884da" %}
+{% set version = "1.4.0.pre" %}
+{% set commit = "e2814ccb224a01289ad3a736ce89a956cb7d52fb" %}
+{% set sha256sum = "55a507878f0d53b8b8319f826df2acd9e52655db317ce04329b9ba3c3880d562" %}
 
 # NOTE(wesm): When updating the build, please remember also to update the
```

Comment:
@wesm I think this comment is outdated, we don't seem to compile Arrow anymore in this build.

Reply:
That's right